opslevel/datasource_opslevel_tier.go

```diff
@@ -52,32 +93,15 @@ func (d *TierDataSource) Schema(ctx context.Context, req datasource.SchemaReques
 	resp.Schema = schema.Schema{
 		MarkdownDescription: "Tier data source",
 
-		Attributes: map[string]schema.Attribute{
-			"alias": schema.StringAttribute{
-				MarkdownDescription: "The human-friendly, unique identifier for the tier.",
-				Computed:            true,
-			},
-			"id": schema.StringAttribute{
-				MarkdownDescription: "The unique identifier for the tier.",
-				Computed:            true,
-			},
-			"index": schema.Int64Attribute{
-				MarkdownDescription: "The numerical representation of the tier.",
-				Computed:            true,
-			},
-			"name": schema.StringAttribute{
-				Description: "The display name of the tier.",
-				Computed:    true,
-			},
-		},
+		Attributes: tierAttributes(map[string]schema.Attribute{}),
```

Comment:
```suggestion
		Attributes: tierDatasourceSchemaAttrs,
```
🤔 Looks like `tierAttributes()` can be removed

Reply:
@davidbloss removed